spec/datadog/tracing/contrib/v1_schema_examples.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.shared_examples 'v1 schema test' do
```

Comment:
The shared_examples is fine right now, might need to tweak a bit in the future for wider adoption.